### PR TITLE
Create issues in primary repos + Grammarly run

### PR DESCRIPTION
**Pull request by dmurko**
Ref: https://github.com/niteoweb/operations/issues/773

### CODE REVIEW

File: 2_Operations/user-stories.md [hunk 1]
```diff
@@ -1,66 +1,66 @@
...
 
-The expected outcomes of the user story which is a list of items stating how the world will look once the user story has been completed. This is not the individual task or activities but rather what will the documentation look like, what new tests will we see, what bug no longer occurs because it has been fixed.
+The expected outcomes of the User Story which is a list of items stating how the world will look once the User Story has been completed. This is not the individual task or activities but rather what will the documentation look like, what new tests will we see, what bug no longer occurs because it has been fixed.
 
 ## Story Points
 
-A [Story Point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story. One full-time member on sprint should be able to do 10 Story Points.
+A [Story Point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-Story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story. One full-time member on sprint should be able to do 10 Story Points.
```
Comment (zupo) [inline 2_Operations/user-stories.md]: I doubt this URL changed ...

File: 2_Operations/user-stories.md [hunk 2]
```diff
@@ -20,45 +20,45 @@ Whenever possible, think about the end user (product persona) and how the result
...
 
 ## Pipelines
 
-From initial creation to being worked upon and finally closed, a user story will be moved to different pipelines as it progresses through a sprint. This is a summary of the workflow of a user story:
+From initial creation to being worked upon and finally closed, a User Story will be moved to different pipelines as it progresses through a sprint. This is a summary of the workflow of a User Story:
 
 1. **New Issues** - New work-in-progress (WIP) user stories, also includes bugs or issues.
-1. **Estimation** - Defined user stories, ready for verifying and story point estimation.
-1. **Backlog** - Waiting for inclusion in a sprint. They are adequately defined with assigned story points.
+1. **Estimation** - Defined user stories, ready for verifying and Story point estimation.
```
Comment (marbeann) [inline 2_Operations/user-stories.md]: Consistency. Shouldn't `user stories` be -> `User Stories`?

Comment (dmurko) [reply 2_Operations/user-stories.md]: Yep, search and replace doesn't know plural :)

File: 2_Operations/user-stories.md (hunk 1 above)
Comment (dmurko) [reply 2_Operations/user-stories.md]: oops, searched and replaced too much

**Pull request merged**